syft/pkg/license.go

```diff
@@ -23,11 +24,11 @@ var _ sort.Interface = (*Licenses)(nil)
 // this is different for licenses since we're only looking for evidence
 // of where a license was declared/concluded for a given package
 type License struct {
-	Value          string             `json:"value"`
-	SPDXExpression string             `json:"spdxExpression"`
-	Type           license.Type       `json:"type"`
-	URLs           internal.StringSet `hash:"ignore"`
-	Locations      file.LocationSet   `hash:"ignore"`
+	Value          string
```

Comment:
Did we intentionally lose the explicit JSON field names on this struct? Will `go` automatically change `Value` to `value` when writing this field to JSON? Otherwise this seems to be it changes the JSON schema.

Reply:
This struct is not used in json marshaling, since there is a `syftjson/model.License` struct that covers this. I made a change to the struct that should have caused a marshaling change and was surprised when there wasn't a change in the output. While debugging I discovered this was the reason and removed the struct tags to not mislead future devs.